cmd/create/machinepool/nodepool.go

```diff
@@ -59,6 +57,21 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		os.Exit(1)
 	}
 
+	// Allow the user to select subnet for a single AZ BYOVPC cluster
+	var subnet string
+	if isBYOVPC(cluster) {
+		subnet = getSubnetFromUser(cmd, r, isSubnetSet, cluster)
```

Comment:
NP
Since this function is now used both for `machinepool` and `nodepool` I wonder if we should move the implementation from `machinepool.go` to a helper file in this package.

Reply:
+1 I can add a helper, it makes sense.